src/components/Address.tsx

```diff
@@ -103,7 +100,7 @@ const Address = ({
           color={color}
           numberOfLines={1}
           ellipsizeMode={ellipsizeMode}
-          padding={padding}
+          {...rest}
```

Comment:
is there any way to do this without spreading? I know eslint doesn't like it because it makes it hard to track what params a component expects